### PR TITLE
Feature/medical claim

### PR DESCRIPTION
**Pull request by sboisvert**
This is really a copy paste job and I think we'll create a better system for beta since it's a bit ridiculous.

accident-band-aid-bandages-42230

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -207,5 +207,12 @@
...
+  "Claim medical expenses": "Claim medical expenses",
+  "Enter the total cost of medical expenses for your self, spouse, and dependent children born after 2001": "Enter the total cost of medical expenses for your self, spouse, and dependent children born after 2001",
+  "Learn more about eligible medical expenses": "Learn more about eligible medical expenses",
+  "Eligible medical expenses can include specialized equipment, accessibility devices, etc. For more information, please refer to https://www.canada.ca/en/revenue-agency/services/tax/individuals/topics/about-your-tax-return/tax-return/completing-a-tax-return/deductions-credits-expenses/lines-330-331-eligible-medical-expenses-you-claim-on-your-tax-return.html#mdcl_xpns": "Eligible medical expenses can include specialized equipment, accessibility devices, etc. For more information, please refer to https://www.canada.ca/en/revenue-agency/services/tax/individuals/topics/about-your-tax-return/tax-return/completing-a-tax-return/deductions-credits-expenses/lines-330-331-eligible-medical-expenses-you-claim-on-your-tax-return.html#mdcl_xpns"
+}
```
Comment (pcraig3) [inline locales/en.json]: You could also delete these if you wanted and then they would go in as part of my PR #70

File: routes/deductions/deductions.controller.js [hunk 2]
```diff
@@ -190,6 +215,33 @@ const postDonationsAmount = (req, res) => {
...
+  }
+
+  req.session.deductions.medicalClaim = false
+
+  //Success, we can redirect to the next page
+  return res.redirect(req.body.redirect)
+}
+
+const postMedicalAmount = (req, res) => {
+  req.session.deductions.medicalAmount = req.body.medicalAmount
```
Comment (pcraig3) [inline routes/deductions/deductions.controller.js]: This line here should change now that @katedee's PR is in.

File: routes/deductions/deductions.controller.js [hunk 3]
```diff
@@ -190,6 +215,33 @@ const postDonationsAmount = (req, res) => {
...
+  }
+
+  req.session.deductions.medicalExpenseClaim = false
+
+  //Success, we can redirect to the next page
+  return res.redirect(req.body.redirect)
+}
+
+const postMedicalAmount = (req, res) => {
+  req.session.deductions.medicalExpenseClaimAmount.amount = req.body.medicalAmount
```
Comment (pcraig3) [inline routes/deductions/deductions.controller.js]: Oh, I guess that was pretty easy.

**Comment by sboisvert** [comment]
I think I need to merge master back in for the tests to pass since now I'm touching req.session.deductions.medicalExpenseClaimAmount.amount which doesn't exist in this branch

**Pull request merged**